recipes/pgcli/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pgcli" %}
+{% set version = "1.6.0" %}
+{% set sha256 = "f19a12ec525ef3f21baab7da67d82e1ca99086a9ba06bdf0592b9412be7c09e8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - pgcli=pgcli.main:cli
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pgspecial >=1.8.0
```

Comment:
Normally we split these into separate PRs to avoid race conditions with the CIs. Though it is always possible to restart the builds.